ovos_tts_plugin_piper/engine.py

```diff
@@ -72,8 +73,11 @@ def synthesize(
         phoneme_ids: List[int] = []
 
         for phoneme in phonemes:
-            phoneme_ids.extend(self.config.phoneme_id_map[phoneme])
-            phoneme_ids.extend(self.config.phoneme_id_map[_PAD])
+            if phoneme in self.config.phoneme_id_map:
+                phoneme_ids.extend(self.config.phoneme_id_map[phoneme])
+                phoneme_ids.extend(self.config.phoneme_id_map[_PAD])
+            else:
+                LOG.warning("No id for phoneme: %s", phoneme)
```

Comment:
Should "f" strings be the norm?

Reply:
i find them much more readable yes, in this case this code is copied from piper repo so I kept the syntax